frontend/src/js/entity-history/actions.ts

```diff
@@ -195,13 +200,13 @@ export function useUpdateHistorySession() {
             defaultEntityHistoryParams.sources,
           );
 
-        const csvUrl = resultUrls.find((url) => url.endsWith("csv"));
+        const csvUrl = resultUrls.find((url) => url.label === "CSV");
```

Comment:
Would this even work? Would there always be a `label` === "CSV" for every editor query?

Reply:
I think it should, but reverting this anyways to the `.endswith("csv")` method. I believe that should be more robust